pages/api/statistics/totals.js

```diff
@@ -22,18 +22,17 @@ export async function getTotalStats() {
 
   let dailyStats = [];
   try {
-    dailyStats = await Stats.find({});
+    dailyStats = await Stats.aggregate([{
```

Comment:
Great improvement 👍 this will definitely help with performance 🚀 

I think this variable should be renamed now as the results are no longer daily but the total